catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java

```diff
@@ -570,7 +584,7 @@ public void exitConstant_expression(BqsParser.Constant_expressionContext ctx) {
         @Override
         public void enterDate(BqsParser.DateContext ctx) {
             print(ctx.getText());
-            dateStr = "";
+            dateStr = normalizeSearchString(ctx.getText());
```

Comment:
I was not able to reproduce this, but I did find another error parsing the created date queries.